draft-ietf-quic-transport.md

```diff
@@ -800,7 +800,7 @@ any outstanding data is declared lost, the endpoint SHOULD send a RESET_STREAM
 frame instead of retransmitting the data.
 
 An endpoint SHOULD copy the error code from the STOP_SENDING frame to the
-RESET_STREAM frame it sends, but MAY use any application error code.  An
+RESET_STREAM frame it sends, but can use any application error code.  An
```

Comment:
Drop the "but can..." -- it's not necessary

Reply:
My point here is that removing the MAY is ok, but the semantics are different. "can" is a statement of fact, and it's not necessary to say that an an endpoint _can_ send any error code in addition to what it SHOULD do. Though I don't feel strongly.

If the can is to be retained, I'd suggest flipping the primary and secondary clauses -- "can use any error code, but SHOULD copy"